src/strings/PowerFxResources.en-US.resx

```diff
@@ -5660,13 +5660,17 @@
     <value>Not supported in formula columns.</value>
     <comment>Error Message. Used in formula columns for scenarios where there the full PowerApps support is not available.</comment>
   </data>
+  <data name="FormulaColumns_MathFunctionBadArity" xml:space="preserve">
+    <value>Invalid number of arguments for {0} function: received {1}, expected {2}</value>
+    <comment>Error Message. Used in formula columns for scenarios where an invalid number of arguments is supplied to a math function. The {0} will be the name of the function. The {1} will be the incorrect number of arguments received. The {2} will be the correct number of args expected. </comment>
+  </data>
   <data name="FormulaColumns_ArgNotSupported" xml:space="preserve">
     <value>The {0} argument is not supported for the {1} function in formula columns.</value>
     <comment>Error Message. Used in formula columns for arguments that are not supported there.  The {0} will be the name of the argument that is not supported.  The {1} will be the name of the function.</comment>
   </data>
   <data name="FormulaColumns_ArgTypeNotSupported" xml:space="preserve">
-    <value>This argument can not be passed as this type in formula columns.</value>
-    <comment>Error Message. Used in formula columns for an argument where the type is not supported.</comment>
+    <value>This argument can not be passed as type {0} in formula columns.</value>
```

Comment:
>{0}  [](http://example.com/codeflow?start=51&length=4)

Is this internal breaking ? (ie, do we need to update Dataverse dll + consumers to pass in arg 0 once we update to this nuget)

Reply:
yeah that's why I added a new one- FormulaColumns_ArgumentTypeNotSupported instead of changing the old one.